modules/core/src/test/java/org/apache/ignite/internal/processors/service/ServicePredicateAccessCacheTest.java

```diff
@@ -75,36 +76,39 @@
     public void testPredicateAccessCache() throws Exception {
         final Ignite ignite0 = startGrid(0);
 
-        ignite0.getOrCreateCache(new CacheConfiguration<>(DEFAULT_CACHE_NAME)
-            .setName("testCache")
-            .setAtomicityMode(ATOMIC)
-            .setCacheMode(REPLICATED)
-            .setWriteSynchronizationMode(FULL_SYNC));
+        CacheConfiguration<String, String> cacheCfg  = new CacheConfiguration<>();
```

Comment:
Why this change?

Reply:
I changed the test to get created cache for putting a value for predicate:
`cache.put(ignite0.cluster().localNode().id().toString(), "val");`

This based on deprecated logic, when service processor use internal cache and contained a value for predicate.